Makefile

```diff
@@ -173,9 +176,12 @@ domains-infra-plan: domains domains-infra-init ## Terraform plan for DNS infrast
 domains-infra-apply: domains domains-infra-init ## Terraform apply for DNS infrastructure (zone and front door). Usage: make domains-infra-apply
 	terraform -chdir=terraform/domains/infrastructure apply -var-file config/zones.tfvars.json ${AUTO_APPROVE}
 
-domains-init: bin/terrafile domains-composed-variables set-azure-account
-	./bin/terrafile -p terraform/domains/environment_domains/vendor/modules -f terraform/domains/environment_domains/config/${ENVIRONMENT}_Terrafile
+.PHONY: vendor-domain-modules
+vendor-domain-modules:
+	rm -rf terraform/domains/environment_domains/vendor/modules/domains
+	git clone --depth=1 --single-branch --branch ${TERRAFORM_MODULES_TAG} https://github.com/DFE-Digital/terraform-modules.git terraform/domains/environment_domains/vendor/modules/domains
```

Comment:
Can we write it `git -c advice.detachedHead=false clone...` to suppress the warning message

Reply:
fixed